Disable available update messages as it outputs invalid upgrade process

--- a/dist/pnpm.mjs
+++ b/dist/pnpm.mjs
@@ -142718,6 +142718,7 @@
 // lib/checkForUpdates.js
 import path53 from "node:path";
 async function checkForUpdates(config2) {
+  return;
   const stateFile = path53.join(config2.stateDir, "pnpm-state.json");
   let state;
   try {
